### ISSUE TITLE
Feature request Themed icons and Widgets 

### ISSUE THREAD

**Issue by jammekod** [GrakovNe/lissen-android#95]
First off, I just want to say how much I love the current icon for the app—it looks great! That said, I’ve noticed it doesn’t yet support **themed icons** on Android 15, and it’s starting to stick out a bit on my homescreen (I’m using a Pixel 9 Pro). It’s not a huge deal, but it would be awesome if this could be added to the roadmap at some point.

With Christmas around the corner, I know this might not be the best timing, but I thought I’d throw the idea out there! A themed icon that matches Material You 3 would make it blend so much better with the rest of apps.

Also, while I’m at it, I was wondering if you’ve thought about adding widgets? Some quick-access widgets or something fun/customizable could be a great addition to the app.

Ideas in a nutshell:

1. Themed Icon Support:

Matches the system theme (Material You) for a more cohesive homescreen experience.

2. Widgets:

Could include shortcuts, live updates, or any other features that make sense for the app.

**Comment by GrakovNe** [comment]
Yes, if someone could share themed icons, i'll be happy to release them for GP

**Comment by jammekod** [comment]
I can't design 😔 hope someone who is create to make some cool themed icons and some widgets to go along this fantastic app 👍😎 fingers crossed

**Comment by GrakovNe** [comment]
Seems like there is no activiy, huh

Let is be closed for now as part of grooming

**Comment by GrakovNe** [comment]
Dear Biccus from [Ko-fi](https://ko-fi.com/B0B61EXUE0) created a cool themed app icon

Will released soon

**Comment by GrakovNe** [comment]
https://github.com/GrakovNe/lissen-android/releases/tag/1.4.15

Now app has:
 - themed icon
 - widget
 - shortcut
